modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -233,6 +245,34 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return this._network;
   }
 
+  static blockExplorer: UTXOBlockExplorer = {
+    baseUrl: "https://api.blockchair.com",
+
+    async getAddressUnspent (addressBase58: string): Bluebird<UnspentInfo[]> {
+      // make a call to blockchair apis
+      const url = `${AbstractUtxoCoin.blockExplorer.baseUrl}/${this.getFullName()}/outputs?addresses=${addressBase58}`;
+      try {
+        const res = await request.get(url);
+        if (res.hasOwnProperty('data')) {
```

Comment:
Is there an `else` missing here? What should happen if the condition is not met?